### ISSUE TITLE
Automatically detect team name when watching on website

### ISSUE THREAD

**Issue by BluTree** [erengy/taiga#805]
This could be nice to detect team name when watching from a website.
For example, if i'm watching an episode on crunhyroll website, set the team of the release to "Crunchyroll".

**Comment by erengy** [comment]
Good idea, though it's not possible with the way recognition currently works. I'll see what I can do.
